.gitpod.yml

```diff
@@ -9,7 +9,7 @@ tasks:
       gatsby-dev --scan-once &&
       npx gatsby build &&
       cd ../.. &&
-      touch /tmp/done.txt
+      gp sync-done build
```

Comment:
I was looking for `gp sync-done` but I couldn't find it anywhere on their docs. Could you point me to a  source of these commands?

Reply:
Oh, my bad, this feature isn't actually documented online yet. I'll try to fix that.

The only place where it's currently "documented" is in `gp help`:

```
$ gp help
Command line interface for Gitpod

Usage:
  gp [command]

Available Commands:
  await-port        Waits for a process to listen on a port
  env               Controls user-defined, persistent environment variables.
  forward-port      Makes a port available on 0.0.0.0 so that it can be exposed to the internet
  help              Help about any command
  init              Create a Gitpod configuration for this project.
  open              Opens a file in Gitpod
  preview           Opens a URL in Theia's preview view
  sync-await        Awaits an event triggered using gp sync-done
  sync-done         Notifies the corresponding gp sync-await calls that this event has happened
  url               Prints the URL of this workspace

Flags:
  -h, --help   help for gp

Use "gp [command] --help" for more information about a command.
```